src/Bridges/intervalbridge.jl

```diff
@@ -48,6 +48,30 @@ function MOI.get(model::MOI.ModelLike, a::MOI.ConstraintDual, c::SplitIntervalBr
     return lower_dual > -upper_dual ? lower_dual : upper_dual
 end
 
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintBasisStatus, c::SplitIntervalBridge)
+    lower_stat = MOI.get(model, MOI.ConstraintBasisStatus(), c.lower)
+    upper_stat = MOI.get(model, MOI.ConstraintBasisStatus(), c.upper)
+    if lower_stat == MOI.NONBASIC_AT_LOWER
+        Compat.@warn("Single sided constraints should not have basis status:" *
```

Comment:
I would say `GreaterThan` instead of "Single sided". If a solver relies on the Interval bridge it means that it does not support Interval so you are not thinking about two-sided vs one-sided when you are implementing MOi for this solver, you are maybe not even aware that two-sided constraints exist